executor/runtime/docker/docker.go

```diff
@@ -795,11 +795,11 @@ func cleanContainerName(prefix string, imageName string) string {
 }
 
 // createVolumeContainerFunc returns a function (suitable for running in a Goroutine) that will create a volume container. See createVolumeContainer() below.
-func (r *DockerRuntime) createVolumeContainerFunc(sCfg *runtimeTypes.SidecarContainerConfig, containerName *string) func(ctx context.Context) error {
+func (r *DockerRuntime) createVolumeContainerFunc(serviceName string, sCfg *runtimeTypes.ServiceOpts) func(ctx context.Context) error {
```

Comment:
Why's `serviceName` a separate param here?  Seems like it could be rolled up into `ServiceOpts`, no?

Reply:
Can do.

The important thing is to not mix this with containerName, which is a long mutated string.